src/test/regress/custom_sql_schedule

```diff
@@ -1 +1,3 @@
 test: custom_sql_test
+test: prepared_statements_1 prepared_statements_2 prepared_statements_3
```

Comment:
Would that make sense to somehow categorize these tests? For example the first one seems like it can be categorized as `repartition`, maybe others can be categorized as some other things. If not, when adding a new test, how will we know which file we should choose?

Reply:
you'd just pick a random one and add? Well, I mostly copy & pasted relevant tests from multi schedule to here, so I don't thing this naming approach would work other than `_1`.

Reply:
I am still not 100% sure how useful this will be but we can always change it later, so it seems better to iterate now and think about it later if something is bothering us